nltk/compat.py

```diff
@@ -121,6 +126,131 @@ def load_module(self, name):
 
     sys.meta_path.insert(0, TkinterLoader())
 
+    import pytz
```

Comment:
@kmike: well noted. do you think this extra dependency is a problem? 
@lrnzcig: we definitely need the `pytz` import, right?